packages/docs/content/migration-guides/standardize-theme-colors.mdx

```diff
@@ -1,13 +1,13 @@
 ---
-title: Theme colors normalization update
+title: Standardizing theme colors
```

Comment:
The reason I included the word "update" was to signal that we're talking about a specific update, but I struggled to make this unique. What if we said something like "2023 theme-color standardization" or "v7 theme-color standardization" or "Standardizing theme colors in v7"?

Reply:
What do you think about this, @line47?

Reply:
That sounds great to me and I agree with the clarification! I like "Standardizing theme colors in v7"